packages/service_kit/pubspec.yaml

```
@@ -1,8 +1,8 @@
-name: strata_service_kit
+name: plasma_service_kit
 description: Topl's ready to use toolkit for Flutter on the Topl Blockchain
 version: 0.1.0
-homepage: "https://www.stratalab.xyz"
-repository: https://github.com/StrataLab/strata_sdk_dart
+homepage: "https://www.plasmalab.xyz"
```

Not sure what we want here. Our actual home page is https://www.plasma.to, but the domain we're using for GitHub and pub.dev is plasmalabs.org. (we don't have anything pointing to the root domain currently)

```suggestion
homepage: "https://www.plasma.to"
```